Source/Libraries/GSF.Security/OIDCSecurityProvider.cs

```diff
@@ -106,19 +122,7 @@ private class TokenResponse
         /// </summary>
         /// <param name="username">Name that uniquely identifies the user.</param>
         public OIDCSecurityProvider(string username)
-            : this(username, false, false, false)
-        {
-        }
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="AdoSecurityProvider"/> class.
-        /// </summary>
-        /// <param name="username">Name that uniquely identifies the user.</param>
-        /// <param name="canRefreshData">true if the security provider can refresh <see cref="UserData"/> from the backend data store, otherwise false.</param>
-        /// <param name="canResetPassword">true if the security provider can reset user password, otherwise false.</param>
-        /// <param name="canChangePassword">true if the security provider can change user password, otherwise false.</param>
-        protected OIDCSecurityProvider(string username, bool canRefreshData, bool canResetPassword, bool canChangePassword)
-            : base(username, canRefreshData, canResetPassword, canChangePassword)
+            : base(username, true, false, false)
```

Comment:
Need to change second parameter to `false` if you are removing `RefreshData` implementation